ontology/yaml/resources/LIGHTING/entity_types/GENERALTYPES.yaml

```diff
@@ -14,43 +14,39 @@
 
 LT:
   description: "Luminaire, lighting fixture."
+  is_abstract: true
   implements:
   - EQUIPMENT
-  uses:
-  - dimmer_percentage_command # in % indicating brightness
-
-LPS:
-  description: "Lighting photocell sensor."
-  implements:
-  - MTS
-
-LMS:
-  description: "Lighting motion sensor."
-  implements:
-  - MTS
 
 LTGW:
   description: "Lighting gateway."
-  implements:
-  - EQUIPMENT
-  
-LAC:
-  description: "lighting area controller."
+  is_abstract: true
   implements:
   - EQUIPMENT
 
 LCM:
   description: "Lighting control module."
+  is_abstract: true
   implements:
   - EQUIPMENT
 
 LKP:
   description: "Lighting keypad."
+  is_abstract: true
   implements:
+  - EQUIPMENT
   - USER_INTERFACE
 
-LIGHTING_GROUP:
+LTB: # May need to add to BOS naming standard
```

Comment:
how is it not a UPS?

Reply:
@pisuke would need to weigh in here...

Reply:
Suggested to describe this as a Lighting central battery unit

Reply:
Conclusion:  Add note to fill this out when we have an example